lms/js_config_types.py

```diff
@@ -64,6 +64,8 @@ class APIAssignments(TypedDict):
 class APIStudents(TypedDict):
     students: list[APIStudent]
 
+    pagination: NotRequired[Pagination]
```

Comment:
As with the other entities, this is optional as the new endpoints does return it but the existing one (the ones driving the data tables) won't.